test/BaseLightStepTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Class BaseLightStepTest
+ * @author Josh Wickham
+ * @copyright &copy; 2016 Life360, Inc.
+ * @since 6/10/16
+ */
+abstract class BaseLightStepTest extends PHPUnit_Framework_TestCase
```

Comment:
abstract base test class to add the `peek` so it's not a global function which would require tests being run in a particular way.